semantic/src/site/globals/site.overrides

```diff
@@ -155,23 +155,23 @@ p strong {
 }
 
 // "Introduction" subheader
-#application-area .ui.header .sub.header {
+.ui.header .sub.header {
   margin-top: 11px;
   margin-bottom: 16px;
   font-size: @fontLarge;
   letter-spacing: @letterSpacingSmall;
 }
 
 // Extra font and margin on Steps header on application home/create
-#application-area .steps-header {
+.steps-header {
   font-size: @fontSmall;
   font-weight: @regularWeight;
   margin-top: 0;
   margin-bottom: 5px;
 }
 
 // Steps line (vertical box - layout)
-#application-area .line-vertical-box {
+.line-vertical-box {
```

Comment:
It's a inner vertical alignment (for progress and action) - maybe it can be renamed?

Reply:
> It's for this region, right?
> ![Screen Shot 2021-04-28 at 1 39 11 PM](https://user-images.githubusercontent.com/5456533/116333221-361d1a80-a827-11eb-8937-534ebc47d37f.png)

Yes, but it's used on the ApplicationHome where the SectionsProgress need to show a progress and action for each row (which is the **line vertical alignment**):
https://github.com/openmsupply/application-manager-web-app/blob/ce73e9cfacaddc1463f13cab46090903b8f9f521/src/components/Application/Sections/SectionsProgress.tsx#L59-L65

Reply:
> Sorry to keep rambling on about a fairly minor thing, I just think it'll benefit us in the long run if we follow agree-upon style patterns.

That's fine. It's difficult because I'm not very used to work with this, so I'm learning here. I'll try to comply to what you have described here, but I'm a little suspicious if we really will be re-using classes that much, so I would rather go with option 2.

Reply:
Done: 1d27129